### ISSUE TITLE
Dragging an .aup3 into Audacity should open the project

### ISSUE THREAD

**Issue by NetJohnP** [audacity/audacity#4816]
### Your idea

Should be able to drag a .aup3 file into the app. Note that it gives an error but apparently the file is thought to be opened, as double-clicking on the .aup3 file (already defined as Audacity project and attempts to open in Audacity) gives an error of the project already open. Closing the Audacity application asked if I wanted to save the project (that was not visible in the application).

### Problem to be solved

Support expected action

**Comment by LWinterberg** [comment]
![image](https://github.com/audacity/audacity/assets/87814144/427a4f05-59b4-4a06-be46-c8518d323baf)

yeah, that's not great.

**Comment by petersampsonaudacity** [comment]
This was deliberately blocked by @lllucius when he was implementing the Unitary Project structure (the AUP3 SQLite dtatabase)

It raises a number of questions:

1) do you **Import** the AUP3 adding it to the existing project (as happens when you import an audio file?
Or do you just open a new project for it?

2) If you import it, what do you do about projects with sim-matched settings,  a good example being  **Project Sample rate**?

3) do you offer the user the option of import versus new project?  Would there be a prefs setting for this?

**Comment by crsib** [comment]
I have a faint memory of discussing it. Personally I think it should open the project in a new window

**Comment by petersampsonaudacity** [comment]
> Personally I think it should open the project in a new window

@crsib unless perhaps the user drags&drops the AUP3 into an empty, and non-dirty, project.

Note that drag&drop of an AUP3 project onto an Audacity app desktop icon works fine in opening a new project (even when other projects are currently open)

**Comment by crsib** [comment]
>  into an empty, and non-dirty, project.

Yep!

**Comment by LWinterberg** [comment]
Duplicate of https://github.com/audacity/audacity/issues/4126
